tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -230,44 +230,27 @@ async def test_read_bytes(alice_workspace):
     assert await alice_workspace.read_bytes("/foo/bar") == b""
 
     await alice_workspace.write_bytes("/foo/bar", b"abcde")
-    assert await alice_workspace.read_bytes("/foo/bar", size=3) == b"abc"
```

Comment:
Those tests are no longer relevant, but shouldn't we do them for `WorkspaceFile` instead ?

Reply:
Those tests already exist, see PR #1367